cumulusci/tasks/salesforce/enable_prediction.py

```diff
@@ -0,0 +1,49 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import SalesforceException
+from simple_salesforce.exceptions import SalesforceError
+
+
+class EnablePrediction(BaseSalesforceApiTask):
+    task_docs = """
```

Comment:
Can we add a sentence or two to state what this task does?

Reply:
Addressed in fba3700214bae1ad0ffc1c9780f50a5496eb5e42